docarray/array/storage/weaviate/find.py

```diff
@@ -31,25 +31,50 @@
 
 class FindMixin:
     def _find_similar_vectors(
-        self, query: 'WeaviateArrayType', limit=10, filter: Optional[Dict] = None
+        self,
+        query: 'WeaviateArrayType',
+        filter: Optional[Dict] = None,
+        limit=10,
```

Comment:
please do not change the order of params

Reply:
I changed them back and will be included in the next push